Client/ClientRoom.cpp

```diff
@@ -254,7 +359,8 @@ void ClientRoom::startLobbyLoop(const std::string &pseudo, const std::string &pa
         }
     }
     if (_state != ClientState::ENDED) {
-        _initEcsGameData();
+        _communicatorInstance.get()->sendDataToAClient(_serverEndpoint, nullptr, 0, 10);
```

Comment:
Why it appears ?

Reply:
Because it's for the lobby loop for the multiplayer mod